sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ToPrettyString.scala

```diff
@@ -45,6 +45,10 @@ case class ToPrettyString(child: Expression, timeZoneId: Option[String] = None)
   override protected def leftBracket: String = "{"
   override protected def rightBracket: String = "}"
 
+  override def kvPairSeparator: String = ":"
+
+  override protected def elementSeparator: String = ","
```

Comment:
```suggestion
  override protected def elementSeparator: String = ", "
```
Add space between kv maybe better.

Reply:
Yea, I think `, ` is better, But spark-sql use `,`, Any suggestions? @cloud-fan